monorepo/website/src/pages/news/2024-11-01-autumn-update.mdx

```diff
@@ -0,0 +1,43 @@
+---
+layout: ../../layouts/MdLayout.astro
+title: "Pathoplexus Autumn Update"
+---
+
+<div>
+    <a href="/news">News</a> > <b>Pathoplexus Autumn Update</b>
+</div>
+
+# Two Weeks of Pathoplexus
+
+*By the Pathoplexus Team - 1 November 2024*
+
+Despite only being 2 months old, we at Pathoplexus have been excited to already contribute to discussions on data sharing, and start becoming part of the pathogen sharing community. In addition, we've continued to add new features and improvements to Pathoplexus - still largely powered by volunteer effort and donated time!
+
+As the seasons turn, we wanted to share some of our recent news:
+
+### In the media:
+- [Lancet Microbe's Elena Dalla Vecchia covered Pathoplexus](https://www.thelancet.com/journals/lanmic/article/PIIS2666-5247(24)00263-5/fulltext), highlighting how Pathoplexus is aiming to improve data sharing through openness, credit, and transparency (13 Sept 2024)
+
+### Visitors and data:
+- Since launch, Pathoplexus had ingested XX new sequences from INSDC-member databases since launch
```

Comment:
```suggestion
- Since launch, Pathoplexus has automatically ingested 53 newly released sequences from INSDC-member databases
```

Reply:
I'm excluding some 500 CCHF ones that were newly ingested due to a bug in NCBI virus

Reply:
Also not counted sequence updates, we could have a separate figure for that

Reply:
We have also got 3 PPX WNV sequences, but might not include this in this update

Reply:
Ah I think I thought the number was bigger due to the CCHF ones :(

Reply:
Does your resolving mean you're happy with as is?

Reply:
Yes I think so, even though it's a bit low number! Still, why not include it 🙃

Reply:
Sorry if I didn't pull this change specifically - it is the current draft as you proposed it.